ingestion/examples/airflow/dags/airflow_lineage_operator.py

```diff
@@ -9,44 +9,71 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 """
-You can run this DAG from the default OM installation
-"""
+You can run this DAG from the default OM installation.
 
-from datetime import datetime, timedelta
+For this DAG to run properly we expected an OpenMetadata
+Airflow connection named `openmetadata_conn_id`.
+"""
+from datetime import datetime
 from textwrap import dedent
 
+import requests
+
 # The DAG object; we'll need this to instantiate a DAG
 from airflow import DAG
 
 # Operators; we need this to operate!
 from airflow.operators.bash import BashOperator
 from airflow.operators.python import PythonOperator
 
+from airflow_provider_openmetadata.hooks.openmetadata import OpenMetadataHook
+
 # These args will get passed on to each operator
 # You can override them on a per-task basis during operator initialization
 from airflow_provider_openmetadata.lineage.operator import OpenMetadataLineageOperator
-from metadata.generated.schema.entity.services.connections.metadata.openMetadataConnection import (
-    OpenMetadataConnection,
-)
-from metadata.generated.schema.security.client.openMetadataJWTClientConfig import (
-    OpenMetadataJWTClientConfig,
-)
+
+OM_HOST_PORT = "http://localhost:8585/api"
+OM_JWT = "eyJraWQiOiJHYjM4OWEtOWY3Ni1nZGpzLWE5MmotMDI0MmJrOTQzNTYiLCJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImlzQm90IjpmYWxzZSwiaXNzIjoib3Blbi1tZXRhZGF0YS5vcmciLCJpYXQiOjE2NjM5Mzg0NjIsImVtYWlsIjoiYWRtaW5Ab3Blbm1ldGFkYXRhLm9yZyJ9.tS8um_5DKu7HgzGBzS1VTA5uUjKWOCU0B_j08WXBiEC0mr0zNREkqVfwFDD-d24HlNEbrqioLsBuFRiwIWKc1m_ZlVQbG7P36RUxhuv2vbSp80FKyNM-Tj93FDzq91jsyNmsQhyNv_fNr3TXfzzSPjHt8Go0FMMP66weoKMgW2PbXlhVKwEuXUHyakLLzewm9UMeQaEiRzhiTMU3UkLXcKbYEJJvfNFcLwSl9W8JCO_l0Yj3ud-qt_nQYEZwqW6u5nfdQllN133iikV4fM5QZsMCnm8Rq1mvLR0y9bmJiD7fwM1tmJ791TUWqmKaTnP49U493VanKpUAfzIiOiIbhg"
+AIRFLOW_HOST_API_ROOT = "http://localhost:8080/api/v1/"
+DEFAULT_OM_AIRFLOW_CONNECTION = "openmetadata_conn_id"
+DEFAULT_AIRFLOW_HEADERS = {
+    "Content-Type": "application/json",
+    "Authorization": "Basic YWRtaW46YWRtaW4=",
+}
 
 default_args = {
-    "retries": 1,
-    "retry_delay": timedelta(minutes=5),
+    "retries": 0,
 }
 
+# Create the default OpenMetadata Airflow Connection (if it does not exist)
+res = requests.get(
+    AIRFLOW_HOST_API_ROOT + f"connections/{DEFAULT_OM_AIRFLOW_CONNECTION}",
```

Comment:
could not figure out a better way to have a default OM Airflow connection out of the box